src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color, FigureList.CIRCLE.name());
+        this.radius = radius;
+    }
+
+    public String getFigureData() {
+        return String.valueOf(radius);
+    }
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. It seems like the `getFigureData` method should be a part of a different interface that is specific to providing data about the figure.